gi_core/dispatch/math/vectorized/openmp/gamma_index_range_impl.cpp

```diff
@@ -0,0 +1,602 @@
+#include <math/vectorized/openmp/gamma_index_range_impl.hpp>
```

Comment:
How this file differs from `gi_core/dispatch/math/vectorized/gamma_index_range_impl.cpp` ?

Reply:
different include -> different function is called in functions below (one with openmp support)